x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -158,23 +159,25 @@ export const createLogThresholdExecutor = (libs: InfraBackendLibs) =>
           indices,
           runtimeMappings,
           scopedClusterClient.asCurrentUser,
-          alertFactory
+          alertFactory,
+          startedAt.valueOf()
         );
       }
     } catch (e) {
       throw new Error(e);
     }
   });
 
-async function executeAlert(
+export async function executeAlert(
   ruleParams: CountRuleParams,
   timestampField: string,
   indexPattern: string,
   runtimeMappings: estypes.MappingRuntimeFields,
   esClient: ElasticsearchClient,
-  alertFactory: LogThresholdAlertFactory
+  alertFactory: LogThresholdAlertFactory,
+  timestamp: number
```

Comment:
Similar to the above comment, can we name this such that it describes the meaning of the timestamp? Is it the `evaluationTimestamp`? Or something else?

And in order to have the code grow in an ordered fashion, how about the insert this argument right after the `timestampField`? Or is there a reason why this should definitely be the last argument?

Reply:
Over in #128020 this is called `executionTimestamp`, which indicates the meaning quite well.

Reply:
I changed it to `executionTimestamp`